src/pilot/discover_nodes/discover_nodes.py

```diff
@@ -473,7 +473,7 @@ def is_idrac(client):
     requests_logger.disabled = True
 
     try:
-        doc = client.client.enumerate(DCIM_iDRACCardView)
+        is_ready = client.client.is_idrac_ready()
```

Comment:
Can you change the comment for this method so that it no longer references DCIM_iDRACCardView?

You could replace it with a comment that just says that since the GetRemoteAPIStatus call is vendor specific, if the server responds to that then that is sufficient to determine that it's an iDRAC.

Reply:
Updated the comment and also changed the return value as suggested.